src/Runner.Listener/JobDispatcher.cs

```diff
@@ -346,7 +346,38 @@ private async Task RunAsync(Pipelines.AgentJobRequestMessage message, string orc
                 }
 
                 var term = HostContext.GetService<ITerminal>();
-                term.WriteLine($"{DateTime.UtcNow:u}: Running job: {message.JobDisplayName}");
+
+                string workflowName = string.Empty;
+                if(message.Variables.TryGetValue("system.workflowFilePath", out var workflowPath))
+                {
+                    workflowName = workflowPath.Value.Split('/').LastOrDefault().Trim();
+                }
+
+                string repositoryName = string.Empty;
+                if(message.ContextData.TryGetValue("github", out var context))
+                {
+                    var githubContext = context as Pipelines.ContextData.DictionaryContextData;
+                    if(githubContext.TryGetValue("repository", out var repository))
+                    {
+                        repositoryName = (repository as Pipelines.ContextData.StringContextData).Value.Trim();
```

Comment:
Same here, if these values are null/unexpected we shouldn't crash the runner

Reply:
Done